oscillogram_classification/clustering_application.py

```diff
@@ -153,3 +136,16 @@ def create_processed_time_series_dataset(data_path: str) -> None:
                 print("FAILURE: ground truth (", test_sample_ground_truth,
                       ") does not match most prominent entry in cluster (", most_prominent_entry, ")")
             print("-------------------------------------------------------------------------")
+
+            if measurement_ids[i] in classification_per_measurement_id:
+                classification_per_measurement_id[measurement_ids[i]][0].append(most_prominent_entry)
+                classification_per_measurement_id[measurement_ids[i]][1].append(test_sample_ground_truth)
+
+            else:
+                classification_per_measurement_id[measurement_ids[i]] = [
```

Comment:
format like this:
![image](https://github.com/tbohne/oscillogram_classification/assets/19288735/c06d530a-cda0-4458-8fd9-da839b3261ef)